product_secondary_unit/models/product_template.py

```diff
@@ -16,3 +16,7 @@ class ProductTemplate(models.Model):
         comodel_name='product.secondary.unit',
         string='Default unit sale',
     )
+    purchase_secondary_uom_id = fields.Many2one(
```

Comment:
@sergio-teruel Why do we have a purchase field into an addon that does not depend on 'purchase'?  This field should be declared into `purchase_order_secondary_unit`

Reply:
The same comment applies to `sale_secondary_uom_id` This one should be declared into `sale_order_secondary_unit`. Do I miss something?

Reply:
I desagree with that. As yo see here (https://github.com/OCA/OCB/blob/11.0/addons/product/models/product_template.py#L104) is in produt module where is declared this field, so I have done the same..

Reply:
I'll not block but I also find not logic to have these 'sale' ant 'purchase' fields of odoo core declared into product. If we can avoid the same mistake...